vocode/streaming/streaming_conversation.py

```diff
@@ -317,6 +318,7 @@ def __init__(
         per_chunk_allowance_seconds: float = PER_CHUNK_ALLOWANCE_SECONDS,
         events_manager: Optional[EventsManager] = None,
         logger: Optional[logging.Logger] = None,
+        output_to_speaker: bool = False,
```

Comment:
I think this should only be a feature for phone calls - way less chance of echo that way